code/modules/hydroponics/gene_modder.dm

```diff
@@ -40,13 +41,27 @@
 	QDEL_NULL(disk)
 	return ..()
 
-/obj/machinery/plantgenes/RefreshParts()
-	rating = 0
-	for(var/I in component_parts)
-		if(istype(I, /obj/item/weapon/stock_parts))
-			var/obj/item/weapon/stock_parts/S = I
-			rating += S.rating-1
-	max_extract_pot = initial(max_extract_pot) + rating*5
+/obj/machinery/plantgenes/RefreshParts() // Comments represent the max you can set per tier, respectively. seeds.dm [219] clamps these for us but we don't want to mislead the viewer.
+	for(var/obj/item/weapon/stock_parts/manipulator/M in component_parts)
+		if(M.rating > 3)
+			max_potency = 95
+		else
+			max_potency = initial(max_potency) + (M.rating**3) // 53,59,77,95 	 Clamps at 100
```

Comment:
For `rating**3`

`1`: 51
`2`: 58
`3`: 77
`4`: 95

For `3**rating`:
`1`: 53
`2`: 59
`3`: 77
`4`: 95

Mars is correct